.idea/runConfigurations/_template__of_mocha_javascript_test_runner.xml

```diff
@@ -8,10 +8,10 @@
       <env name="E2E_RUNNER" value="true" />
       <env name="NODE_ENV" value="test" />
       <env name="TS_NODE_COMPILER_OPTIONS" value="{&quot;strictNullChecks&quot;: false}" />
-      <env name="NOVU_ENTERPRISE" value="true" />
+      <env name="NOVU_ENTERPRISE" value="true      " />
     </envs>
     <ui />
-    <extra-mocha-options>--timeout 30000 --require ts-node/register --exit --file e2e/setup.ts</extra-mocha-options>
+    <extra-mocha-options>--require ts-node/register --exit --file e2e/setup.ts</extra-mocha-options>
```

Comment:
i wonder what is the default timeout, it does not exist it would be better to have it defined here